### PR TITLE
Added better response for organisations

### PR DESCRIPTION
**Pull request by Jnegrier**
Added new endpoint `/api/v1/organization/seo_name/{seo_name}`

Also, this endpoint, and `/api/v1/organization/{id}` will return this new response:

```
{
  "id": 0,
  "name": "",
  "seo_name": "",
  "address": "",
  "mission": "",
  "vision": "",
  "main_contact": {
    "id": 8,
    "firstName": "",
    "lastName": "",
    "photoIdURL": "",
    "countryCollaborator": {
      "countryISO": "",
      "countryISO3": "",
      "callingCode": "",
      "name": ""
    },
    "type": {
      "id": 1,
      "name": ""
    }
  },
  "verified": false,
  "created": "",
  "updated": null,
  "country": ""
}
```

Refactored payload to create a beneficiary `/api/v1/beneficiaries`
(Is still missing data like disabilities, street situation, and organization related)

```
{
  "address": "string",
  "country_iso": "string",
  "date_of_birth": "",
  "first_name": "string",
  "gender": "string",
  "height": 0,
  "last_name": "string",
  "weight": 0
}
```

### CODE REVIEW

File: src/main/java/org/mongen/core/controller/OrganizationController.java [hunk 1]
```diff
@@ -26,9 +27,18 @@
...
-		return ResponseEntity.status(HttpStatus.OK).body(bene);
+	public ResponseEntity<OrganizationResponse> getOrganizationId(@PathVariable("id") Long id){
+		Organization org = organizationServ.findOrganizationById(id);
+		OrganizationResponse org_resp = organizationServ.generateOrganizationResponse(org);
+		return ResponseEntity.status(HttpStatus.OK).body(org_resp);
+	}
+
+	@ApiOperation(value = "Get one Organization by SEO name")
+	@GetMapping("/organization/seo_name/{seo_name}")
+	public ResponseEntity<OrganizationResponse> getOrganizationSeoName(@PathVariable("seo_name") String seo_name){
```
Comment (opishcheiko) [inline src/main/java/org/mongen/core/controller/OrganizationController.java:38]: 😻  thank you!!!

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -1,5 +1,5 @@
 server.port=8080
-spring.datasource.url=jdbc:postgresql://db:5432/mongen_db
+spring.datasource.url=jdbc:postgresql://localhost:5432/mongen_db
```
Comment (opishcheiko) [inline src/main/resources/application.properties]: would this be related to the docker issue?

**Review by opishcheiko — APPROVED** [review]
THaaaaaaaaaanks for such a quick fix! 🧞‍♂️ 
And finding org by name - 💘

File: src/main/resources/application.properties (hunk 2 above)
Comment (Jnegrier) [reply src/main/resources/application.properties]: ops, this change shouldn't be here, the realtime compilation is not working, each time is needed to do a `mvn clean install -DskipTests` to reload, in one moment I lost db connection, so I changed the URL like is shown here. I will continue looking at what the issue with hot reload is

**Pull request merged**